README.md

```diff
@@ -0,0 +1,13 @@
+META-PELUX-RPI-BSP
+====================
+Yocto layer adding configurations required when building PELUX towards raspberry pi.
```

Comment:
Not sure what are the capitalization rules but would prefer "Raspberry Pi" instead of all small caps.